whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -143,6 +111,9 @@ private RdapObject getRdapObject(final String requestUrl, final RpslObject rpslO
             case AUT_NUM:
                 rdapResponse = createAutnumResponse(rpslObject);
                 break;
+            case AS_BLOCK:
```

Comment:
Make sure that it's not possible to query for an as-block, only for the autnum.

See RFC7482 for the full list of allowed types: https://datatracker.ietf.org/doc/rfc7482/?include_text=1

Reply:
If user tries to query specifically for AS_BLOCK, controller will throw Bad Request Exception.
Query to AS_Block if there is no autnum entry is found. In that case we will return AS_Block

Reply:
Great, thanks. Can you add a test that a query for an as-block returns a bad request?

Reply:
Found the as-block test that returns 400 bad request, thanks